packages/discovery-react-components/src/components/SearchFacets/utils/buildAggregationQuery.ts

```diff
@@ -1,12 +1,15 @@
-import DiscoveryV2 from 'ibm-watson/discovery/v2';
+import { QueryAggregationWithName } from './searchFacetInterfaces';
 
-export const buildAggregationQuery = (
-  configuration: DiscoveryV2.QueryTermAggregation[]
-): string => {
-  const aggregation = configuration.map(term => {
-    return term.count
-      ? 'term(' + term.field + ',count:' + term.count + ')'
-      : 'term(' + term.field + ')';
+export const buildAggregationQuery = (configuration: QueryAggregationWithName[]): string => {
+  const aggregation = configuration!.map(term => {
```

Comment:
i don't think you need `configuration!` (just remove the `!`) -> i would say this is in general dangerous to use `!` outside of tests